src/common-service/src/main/java/org/apache/kylin/rest/cache/RedisCache.java

```diff
@@ -128,7 +127,6 @@ public static void getRedisClusterInfo() {
                     slaveNodes.append(host).append(" ");
                 } else {
                     masters.add(host);
-                    masterNodes.append(host).append(" ");
```

Comment:
logger.info("redis cluster slave node: {}", slaveNodes); 

The log uses slaveNodes info, so we can't delete it.

Reply:
I removed only unused masterNodes